recipe/recipe.yaml

```diff
@@ -37,71 +33,97 @@ source:
         - fix-argminmax.patch
 
 build:
-  number: 0
-  skip: not (match(python, python_min ~ ".*") and is_abi3)
-  python:
-    version_independent: true
+  number: 1
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
+      # Since this meta package is noarch: generic, we only need to build it
+      # for one target platform (does not matter which one)
+      skip: target_platform != 'linux-64'
+      noarch: generic
+      string: py${{ python | version_to_buildstring }}h${{ hash }}_${{ polars_variant | replace('-', '_') }}_${{ build }}
```

Comment:
i wouldn't add `py` as a prefix for this package as it's not really a python package but a noarch generic one